From b005ce413078f5ea11cf4d8a2b7de267f0029ab9 Mon Sep 17 00:00:00 2001
From: Andrey Albershteyn <aalbersh@redhat.com>
Date: Mon, 5 Jun 2023 20:34:48 +0200
Subject: [PATCH] fix(nix): make doesn't have enough permission to change
 sources

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c12df98db..47bf34410 100644
--- a/Makefile
+++ b/Makefile
@@ -109,7 +109,7 @@ endif
 
 configure: configure.ac
 	libtoolize -c -i -f
-	cp include/install-sh .
+	cp -f include/install-sh .
 	aclocal -I m4
 	autoconf
 
-- 
2.38.4

